assets/src/data/model/datetime/index.js

```diff
@@ -89,22 +91,24 @@ export const whereConditions = ( {
 	month = 'none',
 } ) => {
 	const where = [];
-	const GREATER_AND_EQUAL = encodeURIComponent( '>=' );
-	const LESS_AND_EQUAL = encodeURIComponent( '<=' );
-
 	if ( ! showExpired ) {
-		where.push( 'where[DTT_EVT_end**expired][]=>&where[DTT_EVT_end**expired][]=' +
-			nowDateAndTime.local().format() );
+		where.push(
+			'where[DTT_EVT_end**expired][]=' + GREATER_THAN +
+			'&where[DTT_EVT_end**expired][]=' +
+			nowDateAndTime.local().format()
+		);
 	}
 	if ( month && month !== 'none' ) {
-		where.push( 'where[DTT_EVT_start][]=' +
-			GREATER_AND_EQUAL +
+		where.push(
+			'where[DTT_EVT_start][]=' + GREATER_THAN +
 			'&where[DTT_EVT_start][]=' +
-			moment().month( month ).startOf( 'month' ).local().format() );
-		where.push( 'where[DTT_EVT_end][]=' +
-			LESS_AND_EQUAL +
+			moment().month( month ).startOf( 'month' ).local().format()
+		);
+		where.push(
+			'where[DTT_EVT_end][]=' + LESS_THAN +
 			'&where[DTT_EVT_end][]=' +
```

Comment:
I think this should remain `LESS_AND_EQUAL` here because we want to include the last day of the month in the query right?